packages/validator-bonds-cli-core/src/commands/manage/initWithdrawRequest.ts

```diff
@@ -72,6 +72,12 @@ export function configureInitWithdrawRequest(program: Command): Command {
       'Rent payer for the account creation (default: wallet keypair)',
       parseWalletOrPubkey,
     )
+    .option(
+      '--compute-unit-limit <number>',
+      'Compute unit limit for the transaction (default value based on the operation type)',
```

Comment:
```suggestion
      'Compute unit limit for the transaction',
```
(non-blocking)
This seems to have a specific default value of `INIT_WITHDRAW_REQUEST_LIMIT_UNITS` instead of being based on the operation type, so this might be a copy-paste error.